### ISSUE TITLE
Move Nginx configuration to use the nginx formula

### ISSUE THREAD

**Issue by blarghmatey** [mitodl/elasticsearch-formula#17]
This formula was written before we started using the nginx-formula so it has some custom routines to manage the nginx process and configuration. We should remove those pieces in favor of pillar-driven nginx configs to support proxying to Kibana.

**Comment by blarghmatey** [comment]
The elasticsearch-formula is now deprecated so this issue is no longer relevant.
